app/pages/docs/tutorial.mdx

```diff
@@ -691,19 +696,19 @@ export default resolver.pipe(
 )
 ```
 
-#### Try it out
+#### Provalo
 
-Now you can go to `localhost:3000/questions/new` and create a new question
-with choices!
+Adesso puoi collegarti su `localhost:3000/questions/new` e creare una
+nuova domanda completa di scelte!
 
-## Listing choices {#listing-choices}
+## Elencare le choice {#listing-choices}
```

Comment:
Avendo tradutto `choices` fino ad ora penso sia meglio tradurre anche qui.

Reply:
Si mi è sfuggito. Ero indeciso in effetti se tradurre choice e question; all'inizio non l'avevo fatto ma in alcuni punti suonava veramente male.